apps/editor/src/app/editor-app.component.ts

```diff
@@ -161,6 +164,12 @@ export class EditorAppComponent {
     });
   }
 
+  ngAfterViewInit() {
```

Comment:
Why do you use AfterViewInit?

Reply:
I decided to use this lifecycle hook to avoid [ExpressionChangedAfterItHasBeenCheckedError](https://github.com/angular/angular/issues/17572).